handbook/engineering/README.md

```diff
@@ -632,6 +632,8 @@ If the bug does not meet the criteria of a critical bug, the EM will determine i
 
 When fixing the bug, if the proposed solution requires changes that would affect the user experience (UI, API, or CLI), notify the EM and PM to align on the acceptability of the change.
 
+Engineering teams collaboratively handle bug fixes. If one team is at capacity and a bug needs attention, another team can step in to assist. When the MDM team helps with a CX bug, use the `~leaning on MDM` label. If the CX team helps with an MDM bug, use the `~leaning on CX` label.
```

Comment:
Is there some consequence of using that label? Does it appear in this or that board?

Reply:
@zhumo added clarification